clustered/integration-test/src/test/java/org/ehcache/clustered/management/AbstractClusteringManagementTest.java

```diff
@@ -199,92 +168,46 @@ protected void initCM() throws InterruptedException {
     );
   }
 
-  public static void initIdentifiers() throws Exception {
-    tmsServerEntityIdentifier = null;
-    ehcacheClientIdentifier = null;
-    clusterTierManagerEntityIdentifier = null;
-
-    do {
-      tmsServerEntityIdentifier = readTopology()
-        .activeServerEntityStream()
-        .filter(serverEntity -> serverEntity.getType().equals(NmsConfig.ENTITY_TYPE))
-        .filter(AbstractManageableNode::isManageable)
-        .map(ServerEntity::getServerEntityIdentifier)
-        .findFirst()
-        .orElse(null);
-      sleep(500);
-    } while (tmsServerEntityIdentifier == null && !Thread.currentThread().isInterrupted());
-
-    do {
-      ehcacheClientIdentifier = readTopology().getClients().values()
+  private static void initIdentifiers() throws Exception {
+    while ((ehcacheClientIdentifier = readTopology().getClients().values()
```

Comment:
It would be nice to have a version of WaitAssert that is able to return the value from the callable that has completed with the assertion inside, something like:
`ehcacheClientIdentifier = assertThat(() -> { return ... }, is(notNull()))`